cgmanifest.json

```diff
@@ -189,6 +189,78 @@
             }
          }
       },
+      {
```

Comment:
You may remove dotnet/corefxlab and dotnet/standard

Reply:
I believe these are needed/used to generate the C# packages.